sdk/communication/communication-phone-numbers/src/phoneNumbersClient.ts

```diff
@@ -106,10 +103,9 @@ export class PhoneNumbersClient {
     const authPolicy = createCommunicationAuthPolicy(credential);
     this.client.pipeline.addPolicy(authPolicy);
 
-    // These policies are temporary workarounds to address compatibility issues with Azure Core V2.
+    // This policy is temporary workarounds to address compatibility issues with Azure Core V2.
```

Comment:
As per this comment, I believe that the pagination workaround is also temporary, right?

Reply:
To answer both of your comments, the intention is for it to be a temporary solution. However, changing our API to use absolute URLs is probably not going to be done in the short term, if at all; thus there is no exact timeline for when this workaround will be removed.